pinc/theme.inc

```diff
@@ -119,30 +119,96 @@ function html_logobar()
 {
     global $code_url, $site_name;
     global $maintenance, $testing, $alert_message;
+    global $blog_url, $wiki_url;
 
     echo "\n<div id='header'>\n";
-    echo "<a href='$code_url'>\n";
+
     echo "<div id='logo'>\n";
-    echo "<img src='$code_url/graphics/dp-logo.svg' alt='$site_name'>\n";
-    echo "</div>\n";
+    echo "<a href='$code_url'>\n";
+    echo "<img id='logo-img' src='$code_url/graphics/dp-logo.svg' alt='$site_name'>\n";
```

Comment:
I wonder do we want to escape $site_name? It could contain single quotes, e.g. "Brian's Proofing Emporium".

Reply:
Good catch - fixed this and the other instance in this file.